includes/class-sensei-course.php

```diff
@@ -378,11 +378,18 @@ public function course_video_meta_box_content () {
 
 		$course_video_embed = get_post_meta( $post->ID, '_course_video_embed', true );
 
+		if ( Sensei_Utils::html_security() ) {
```

Comment:
I see this pattern in lots of places

```
if (security is enabled) {
    return kses(input);
}

return input;
```

I would refactor this into a function, call it something like this

`Sensei_Wp_Kses::maybe_sanitize( $content, $allowed_html )`

It would take the content string and the allowed html array and do the option check